### ISSUE TITLE
Experiments to check validity of model

### ISSUE THREAD

**Issue by Samantha-Zhan** [YilikaLoufoua/noise-suppression#16]
Needs Giancarlo's assistant to run PyTorch experience and see the expected training rate.

**Comment by Samantha-Zhan** [comment]
1. We will be running a larger experiment with `batch size: 48` and train with 300 clean&noisy data linked [here](https://drive.google.com/file/d/19nxDJYNfrZaFJa0uMjsA_sLLMBr-TkIB/view?usp=sharing). Please provide result data on the **speed of training(mins per iteration)** and **quality of training(loss and RMSE)** after **100** iterations.
2. Then, work with a smaller batch size `batch size: 5` and train with 50 clean&noisy data linked [here](https://drive.google.com/file/d/1nAEE4XhLcjA12THqAWnNVc84hdzI8Orb/view?usp=sharing). Please finish the training and provide result data on **total duration of training** and **training result** (graph/numerical values)
